dayrepo/snippets/templates/car_post.html

```diff
@@ -0,0 +1,26 @@
+<head>
+    {% load static %}
+    <link rel="stylesheet" href='{% static "css/cars.css" %}'>
+</head>
+<h2>車両情報設定</h2>
+<form method="post">
+    <p>
+    <div id=vehicle_number>
+        <label>車両番号:</label>
+        <input type="text"  maxlength="10" id="place_name" name="place_name" placeholder="札幌" required>
+        <input type="number" maxlength="3" id="class_number" name="class_number" placeholder="100" required>
+        <input type="text" maxlength="1" id="kana" name="kana" placeholder="あ" required>
+        <input type="number" maxlength="2" id="serial_number_top" name="serial_number_top" placeholder="88">
+        -
+        <input type="number" maxlength="2" id="serial_number_end" name="serial_number_end" placeholder="88" required>
+    </div>
+    </p>
+    <p>
+        <label for="id_now_mileage">走行距離:</label>
+        <input type="number" id="now_mileage" name="now_mileage" placeholder="車両の初期メーター値km" required>    
```

Comment:
特に理由がなければ、nameは基本的に不要です
```suggestion
        <input type="number" maxlength="3" id="class_number" placeholder="100" required>
        <input type="text" maxlength="1" id="kana" placeholder="あ" required>
        <input type="number" maxlength="2" id="serial_number_top" placeholder="88">
        -
        <input type="number" maxlength="2" id="serial_number_end" placeholder="88" required>
    </div>
    </p>
    <p>
        <label for="id_now_mileage">走行距離:</label>
        <input type="number" id="now_mileage" placeholder="車両の初期メーター値km" required>    
```